### PR TITLE
feat: Remove the 'Connect Wallet' button from the Hero section

### LINKED ISSUE #58: Remove the Connect Wallet button in hero section
## 📘 Description

The "Connect Wallet" button in the Hero section on the Landing Page should be removed, as it has now been moved to the header for better navigation and visibility.

## 🔢 Steps

1. Remove the "Connect Wallet" button from the Hero section.
2. Reposition the "Explore Marketplace" button for better alignment and design.
3. Ensure the "Explore Marketplace" button is responsive.

## ✅ Acceptance Criteria

- The "Connect Wallet" button has been removed from the Hero section.
- The "Explore Marketplace" button is properly aligned and visually appealing.
- The button is responsive.

## 🌎 References

Not applicable

## 📜 Additional Notes

⚠ **Do not apply until the ODHack begins.**
⚠ **Read our guidelines before applying.**

### PR DESCRIPTION
**Pull request by Jagadeeshftw**
# 📝 Pull Request Title
- feat: Remove the 'Connect Wallet' button from the Hero section

## 🛠️ Issue
- Closes #58 

## 📖 Description
- This PR addresses issue #58 by removing the "Connect Wallet" button from the Hero section of the Landing Page. This change aligns with the updated design decision to move the "Connect Wallet" button to the header for better navigation and visibility. No changes were needed for the "Explore Marketplace" button, as it is already responsive and properly aligned.

## ✅ Changes made
- Removed the "Connect Wallet" button from the Hero section.
- Verified the responsiveness and alignment of the "Explore Marketplace" button.

## 🖼️ Media (screenshots/videos)
-
image

-
image

## 📜 Additional Notes
-

### CODE REVIEW

**Comment by Jagadeeshftw** [comment]
Can you review this @danielcdz ?

File: frontend/app/components/ui/hero-section.tsx [hunk 1]
```diff
@@ -39,10 +39,6 @@ export function HeroSection() {
 				</p>
 
 				<div className="flex flex-col sm:flex-row gap-4 justify-center items-center">
-					<Button size="lg" className="group">
-						<Wallet className="mr-2 h-4 w-4 transition-transform group-hover:scale-110 dark:text-white" />
-						Connect Wallet
-					</Button>
 					<Link href="/marketplace">
 						<Button variant="outline" size="lg" className="group">
```
Comment (derianrddev) [inline frontend/app/components/ui/hero-section.tsx]: On this button, remove the `variant` property, and from the icon, remove `dark:text-white`.

File: frontend/app/components/ui/hero-section.tsx [hunk 2]
```diff
@@ -63,4 +59,4 @@ export function HeroSection() {
 			</div>
 		</div>
 	);
-}
+}
```
Comment (derianrddev) [inline frontend/app/components/ui/hero-section.tsx]: Can you add a line at the end of the file?

**Review by derianrddev — CHANGES_REQUESTED** [review]

**Comment by derianrddev** [comment]
@Jagadeeshftw Good job! Just some minor comments. Please update the screenshot once you make the changes.

**Comment by Jagadeeshftw** [comment]
@derianrddev your review comments were addressed and the screenshot was also updated.

**Review by derianrddev — APPROVED** [review]

**Review by danielcdz — APPROVED** [review]
LGTM! thanks for your work

**Pull request merged**